src/character.cpp

```diff
@@ -11631,53 +11632,76 @@ void Character::pause()
         try_remove_grab();
     }
 
-    // put pressure on bleeding wound, prioritizing most severe bleeding
+    // put pressure on bleeding wound, prioritizing most severe bleeding that you can compress
     if( !is_armed() && has_effect( effect_bleed ) ) {
+        // Calculate max staunchable bleed level
+        // Top out at 20 intensity for base, unencumbered survivors
+        int max = 20;
+        max *= get_modifier( character_modifier_bleed_staunch_mod );
+        add_msg_debug( debugmode::DF_CHARACTER, "Staunch limit after limb score modifier %d", max );
+
+        // +5 bonus if you know your first aid
+        if( has_proficiency( proficiency_prof_wound_care ) ||
+            has_proficiency( proficiency_prof_wound_care_expert ) ) {
```

Comment:
why is this an OR argument if you inherently must have Wound Care mastered to even get Wound Care Expert 🤔

Reply:
Just a failsafe in case anything (debug commands or EoCs) teach you one skipping the pereqs.